src/measure_theory/group/fundamental_domain.lean

```diff
@@ -326,33 +334,55 @@ and_congr (hs.ae_strongly_measurable_on_iff ht hf) (hs.has_finite_integral_on_if
 
 variables [normed_space ℝ E] [complete_space E]
 
+@[to_additive] lemma integral_eq_tsum_of_ac (h : is_fundamental_domain G s μ) (hν : ν ≪ μ)
```

Comment:
This lemma doesn't seem really natural to me, as it mixes two orthogonal things: absolutely continuous measures, and integrals. I'd rather have a lemma saying that, if `s` is a fundamental domain for mu, then it is also a fundamental domain for nu is nu is absolutely continuous with respect to mu. And another lemma which is just `integral_eq_tsum`. In applications, you can chain the two with dot notation if needed.

Reply:
This was just meant to be a mutatis mutandis copy of `lintegral_eq_tsum_of_ac`. Do you recommend changing how that functions as well?

Reply:
This is actually not possible. We looked into it but we can't make that change, because `set_lintegral_eq_tsum` requires that there be an `smul_invariant_measure` mu that is restricted (which is an absolutely continuous measure w.r.t. mu); the restriction is not itself smul invariant.

Reply:
Then

maintainer merge